src/main/kotlin/javabot/service/UrlCacheService.kt

```diff
@@ -0,0 +1,39 @@
+package javabot.service
+
+import com.google.common.cache.CacheBuilder
+import com.google.common.cache.CacheLoader
+import com.google.common.cache.LoadingCache
+import com.google.inject.Inject
+import com.google.inject.Singleton
+import javabot.JavabotConfig
+import net.swisstech.bitly.BitlyClient
+import javax.annotation.Nullable
+
+@Singleton
+class UrlCacheService @Inject constructor(private val config: JavabotConfig) {
+    fun shorten(url: String): String {
+        return urlCache[url]
+    }
+
+    @field:[Nullable Inject(optional = true)]
```

Comment:
@Nullable is redundant, yeah?

Reply:
Yeah, it was cargo culted from the original URL shortening. I didn't want to change something when I wasn't sure what the implications were. Removing.